src/geo/projection/globe.js

```diff
@@ -44,9 +44,9 @@ export default class Globe extends Mercator {
     }
 
     projectTilePoint(x: number, y: number, id: CanonicalTileID): {x: number, y: number, z: number} {
-        const tiles = Math.pow(2.0, id.z);
-        const mx = (x / EXTENT + id.x) / tiles;
-        const my = (y / EXTENT + id.y) / tiles;
+        const tileScale = 1 << id.z;
+        const mx = (x / EXTENT + id.x) / tileScale;
+        const my = (y / EXTENT + id.y) / tileScale;
```

Comment:
`tileScale` seems like an incorrect name to use, no? Since `1 << id.z` represents the number of tiles for zoom level `id.z`, either `tiles`/`numTiles`/`tilesCount` would be more representative of what this value is.